src/main/java/io/github/thebusybiscuit/exoticgarden/items/BonemealableItem.java

```diff
@@ -0,0 +1,23 @@
+package io.github.thebusybiscuit.exoticgarden.items;
+
+import io.github.thebusybiscuit.slimefun4.api.items.ItemSetting;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+import org.bukkit.inventory.ItemStack;
+
+public class BonemealableItem extends SlimefunItem {
```

Comment:
Not a fan of this name but ehh, can live with that.
Please add javadocs to this class though :)